includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -146,6 +146,9 @@ public function capture_terminal_payment( $request ) {
 					[ 'status' => $http_code ]
 				);
 			}
+			// Store receipt generation URL for mobile applications in order meta-data.
+			$order->add_meta_data( 'receipt_url', get_rest_url( null, 'wc/v3/payments/readers/receipts/' . $intent->get_id() ) );
```

Comment:
The API version `v3` is hardcoded here. Is there a way to dynamically add it here?

Reply:
Good point :eyes:

Reply:
Addressed in 01fed4c4b9947b65bd30c5bd54a7bc2cc3245977